portfolio/src/main/webapp/style.css

```diff
@@ -1,3 +1,4 @@
+@import url(fontawesome.min.css);
```

Comment:
Since you're adding the stylesheet in the HTML, this becomes an extraneous import.

Reply:
Probably going to remove that too